filter.go

```diff
@@ -26,3 +26,65 @@ func extractMetrics(re *regexp.Regexp, in <-chan []byte, out chan<- []byte) {
 
 	close(out)
 }
+
+func SkipLuhnMatches(in <-chan []byte) <-chan []byte {
+	out := make(chan []byte)
+
+	re := regexp.MustCompile(`\D+`)
+	go skipLuhnMatches(re, in, out)
```

Comment:
That's a fair point. I'm not sure why I defined it outside; I think it might have been refactored to end up like that.